wagtail/admin/templates/wagtailadmin/permissions/includes/collection_member_permissions_formset.html

```diff
@@ -45,30 +45,14 @@
         </tbody>
     </table>
 
-    <template id="id_{{ formset.prefix }}-EMPTY_FORM_TEMPLATE">
-        <tr id="inline_child_{{ formset.empty_form.prefix }}">
+    <template id="id_{{ formset.prefix }}-EMPTY_FORM_TEMPLATE" data-w-formset-target="template">
+        <tr id="inline_child_{{ formset.empty_form.prefix }}" data-w-formset-target="child">
             {% include "wagtailadmin/permissions/includes/collection_member_permissions_form.html" with form=formset.empty_form only %}
         </tr>
     </template>
 
     <p class="add">
-        <button class="button bicolor button--icon" id="id_{{ formset.prefix }}-ADD" value="Add" type="button">{% icon name="plus" wrapped=1 %}{% block add_button_label %}{% endblock %}</button>
+        <button class="button bicolor button--icon" id="id_{{ formset.prefix }}-ADD" type="button" data-action="w-formset#add">{% icon name="plus" wrapped=1 %}{% block add_button_label %}{% endblock %}</button>
```

Comment:
```suggestion
        <button class="button bicolor button--icon" id="id_{{ formset.prefix }}-ADD" value="Add"  type="button" data-action="w-formset#add">{% icon name="plus" wrapped=1 %}{% block add_button_label %}{% endblock %}</button>
```

Accidentally removed `value="Add"`. I actually think we should not be setting the value on these buttons as it's not used visually and I'm pretty sure it's not used by the server.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#value

Probably an investigation for a different issue.